mscore/CMakeLists.txt

```diff
@@ -560,13 +560,23 @@ if (MINGW)
 
    install( TARGETS mscore RUNTIME DESTINATION bin )
 
+   if (BUILD_64)
+      install( FILES
+         ${MINGW_ROOT}/bin/libgcc_s_seh-1.dll
+         ${MINGW_ROOT}/lib/libportaudio-x86_64-w64-mingw32.static.dll
```

Comment:
Maybe better separate `install(FILES ...` for these and a
`${MINGW_ROOT}/lib/portaudio.dll RENAME libportaudio-x86_64-w64-mingw32.static.dll`
for portaudio. Need to check and test...

Reply:
OK, tested, it works, so got implemented